com.unity.netcode.gameobjects/Editor/CodeGen/NetworkBehaviourILPP.cs

```diff
@@ -20,6 +20,9 @@ namespace Unity.Netcode.Editor.CodeGen
 
     internal sealed class NetworkBehaviourILPP : ILPPInterface
     {
+        private const string k_ExtensionMethodReadName = "ReadValueSafe";
+        private const string k_ExtensionMethodWriteName = "WriteValueSafe";
```

Comment:
could we use `nameof()` operator here instead of hardcoded strings?

Reply:
Good call, will be fixed in the next commit